api/app/db/crud/hfi_calc.py

```diff
@@ -21,9 +24,50 @@ def get_all_stations(session: Session) -> CursorResult:
     return session.query(PlanningWeatherStation.station_code).all()
 
 
+def get_planning_areas(session, fire_centre_id: int) -> CursorResult:
+    """ Get all planning areas for the supplied fire centre """
+    return session.query(PlanningArea)\
+        .filter(PlanningArea.fire_centre_id == fire_centre_id)\
+        .order_by(PlanningArea.order_of_appearance_in_list)
+
+
+def get_fire_centre_planning_area_stations(session, fire_centre_id: int) -> CursorResult:
+    """ Get all the stations for a planning area """
+    return session.query(PlanningWeatherStation)\
+        .join(PlanningArea, PlanningArea.id == PlanningWeatherStation.planning_area_id)\
+        .filter(PlanningArea.fire_centre_id == fire_centre_id)\
+        .order_by(PlanningArea.id)
```

Comment:
I'm confused by this function - is it returning all stations for a specific planning area or a specific fire centre? The documentation/comment says PA, the method signature suggests FC?

Reply:
Maybe you can think of a better name for it? The relationship is Fire Centre -> Planning Area -> Station - so saying "get fire centre station" is really the same thing as "get fire centre planning area stations" - but less confusing? Should I just go with "get_fire_centre_stations" ?
Obviously have to update the documentation - yes - for sure.

Reply:
I've pushed a change - have a look if that's better?

Reply:
Gotcha. I think the name of the function is fine, I was just confusing by planning area in the documentation, `fire_centre_id` in the parameters

Reply:
Next question - I haven't found yet where this method gets called, but does it make sense to order the results by `PlanningArea.id`? Shouldn't they be getting sorted according to the planning area's order within the fire centre, and each station's order within the planning area? (Obviously this could overlap with my ticket - I just want to get a sense of what extra changes I might need to make in my PR related to this PR)

Reply:
Oh - thanks for spotting that! It was for iterating through with grouped planning areas, but I fixed that - I'll remove the sorting.